kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/resources/kms/aws/KubeAwsKmsCloudTestKmsFacade.java

```diff
@@ -6,93 +6,95 @@
 
 package io.kroxylicious.systemtests.resources.kms.aws;
 
-import java.time.Duration;
-import java.util.List;
+import java.net.URI;
 
-import io.kroxylicious.kms.provider.aws.kms.model.DescribeKeyResponse;
+import io.kroxylicious.kms.provider.aws.kms.AwsKmsTestKmsFacade;
+import io.kroxylicious.kms.provider.aws.kms.config.Config;
+import io.kroxylicious.kms.provider.aws.kms.model.RotateKeyRequest;
 import io.kroxylicious.kms.service.TestKekManager;
-import io.kroxylicious.systemtests.executor.Exec;
-import io.kroxylicious.systemtests.executor.ExecResult;
+import io.kroxylicious.proxy.config.secret.InlinePassword;
 import io.kroxylicious.systemtests.installation.kms.aws.AwsKmsCloud;
-import io.kroxylicious.systemtests.k8s.exception.KubeClusterException;
 
-import static io.kroxylicious.kms.provider.aws.kms.AwsKms.ALIAS_PREFIX;
-import static io.kroxylicious.kms.provider.aws.kms.AwsKmsTestKmsFacade.CREATE_KEY_RESPONSE_TYPE_REF;
-import static io.kroxylicious.kms.provider.aws.kms.AwsKmsTestKmsFacade.DESCRIBE_KEY_RESPONSE_TYPE_REF;
-import static io.kroxylicious.kms.provider.aws.kms.AwsKmsTestKmsFacade.SCHEDULE_KEY_DELETION_RESPONSE_TYPE_REF;
+import edu.umd.cs.findbugs.annotations.NonNull;
 
 /**
  * KMS Facade for AWS Kms Cloud.
  * Uses command line interaction so to avoid the complication of exposing the AWS Cloud endpoint
  * to the test outside the cluster.
  */
-public class KubeAwsKmsCloudTestKmsFacade extends AbstractKubeAwsKmsTestKmsFacade {
-    private final String awsCmd;
-    private String kekKeyId;
+public class KubeAwsKmsCloudTestKmsFacade extends AwsKmsTestKmsFacade {
+    private final AwsKmsCloud awsKmsCloud;
```

Comment:
note to self: take a look at this

Reply:
@k-wall did you have time to look at this?

Reply:
it is fine for now.